recipes/mumps/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "mumps" %}
+{% set version = "5.0.1" %}
+{% set sha256 = "50355b2e67873e2239b4998a46f2bbf83f70cdad6517730ab287ae3aae9340a0" %}
+
+package:
+  name: mumps
+  version: {{ version }}
+
+source:
+  fn: MUMPS_{{ version }}.tar.gz
+  url: http://mumps.enseeiht.fr/MUMPS_{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - toolchain
+    - metis
+    - scotch
+    - openblas 0.2.18|0.2.18.*.
+    - gcc
+  run:
+    - metis
+    - scotch
+    - openblas 0.2.18|0.2.18.*.
+    - libgfortran
+
+test:
+  commands:
+    - test -f "${PREFIX}/lib/libzmumps.a"
+    - test -f "${PREFIX}/lib/libcmumps.a"
+    - test -f "${PREFIX}/lib/libdmumps.a"
+    - test -f "${PREFIX}/lib/libsmumps.a"
+    - test -f "${PREFIX}/lib/libmumps_common.a"
+    - test -f "${PREFIX}/lib/libpord.a"
+    - test -f "${PREFIX}/lib/libmpiseq.a"
+    - test -f "${PREFIX}/include/dmumps_struc.h"
+    - test -f "${PREFIX}/include/zmumps_struc.h"
+    - test -f "${PREFIX}/include/cmumps_struc.h"
+    - test -f "${PREFIX}/include/smumps_struc.h"
```

Comment:
Is there some example code that we can build and test here? Since we don't have `make check`, this will be very helpful.